Documentation/gettingstarted/clustermesh.rst

```diff
@@ -41,6 +41,13 @@ Prerequisites
 * The network between clusters must allow the inter-cluster communication. The
   exact ports are documented in the :ref:`firewall_requirements` section.
 
+.. note::
+
+   By default the global.identityAllocationMode is set to ``crd``. With direct
+   etcd connections, in order for the identities to be propagated across the
+   clusters and for cross-cluster policy to work, the identityAllocationMode
+   should be set to ``kvstore`` as shown below
+   ``--set global.identityAllocationMode=kvstore``.
```

Comment:
Two nits here:

(1) This is also a prerequisite. There's no reason this particular item is special enough to call out in a `note`. Probably it makes sense to expand the bullet point on `Cilium must bec onfigured to use etcd as the kvstore....` to mention both that the `kvstore` option must be configured as well as the `identity-allocation-mode`.

(2) `global.identityAllocationMode` at the start of this paragraph is not encased in double-backticks but it probably makes sense to consistently format references to the configuration objects (ie encase this one too).

Reply:
Ok will address the review comments.

Reply:
Done.